doc/modules/clustering.rst

```diff
@@ -1569,3 +1569,77 @@ Drawbacks
  *  Caliński, T., & Harabasz, J. (1974). "A dendrite method for cluster
     analysis". Communications in Statistics-theory and Methods 3: 1-27.
     `doi:10.1080/03610926.2011.560741 <http://dx.doi.org/10.1080/03610926.2011.560741>`_.
+
+.. _davies–bouldin_index:
+
+Davies–Bouldin Index
+----------------------
+
+If the ground truth labels are not known, the Davies–Bouldin index
+(:func:`sklearn.metrics.davies_bouldin_index`) can be used to evaluate the
```

Comment:
You need an entry in classes.rst for this to work. You should have an entry there anyway!